TestBenches/ProjectionCalculator_test.cpp

```diff
@@ -59,14 +57,12 @@ int main()
   TrackletProjectionMemory<BARRELPS> tproj_barrel_ps[TC::N_PROJOUT_BARRELPS];
   TrackletProjectionMemory<BARREL2S> tproj_barrel_2s[TC::N_PROJOUT_BARREL2S];
   TrackletProjectionMemory<DISK> tproj_disk[TC::N_PROJOUT_DISK];
+  TrackletParameterMemory trackletParametersOut;
 
   // print the input files loaded
-  std::cout << "Loaded the input files:\n";
 
   // loop over events
-  cout << "Start event loop ..." << endl;
   for (unsigned int ievt = 0; ievt < nevents; ++ievt) {
-    cout << "Event: " << dec << ievt << endl;
```

Comment:
@aryd I'd recommend keeping at least the printout of the event number. That can be useful for debugging.

Reply:
Restored.